manifests/script.pp

```diff
@@ -27,10 +27,10 @@
 # content param in the file type.
 #
 define rclocal::script (
+  String[1]                 $content,
   Enum['absent', 'present'] $ensure   = 'present',
   String[1]                 $priority = '50',
   Boolean                   $autoexec = true,
-  String                    $content  = '',
```

Comment:
It does make sense or at least is unnescsary if `ensure => absent`.

I don't use this module however I do do:

```
if versioncmp($facts['os']['release']['major'],'9') >= 0 {
  file{'/etc/rc.d/rc.local':                              
    ensure => absent,
  }
}
```
on EL9 other wise you get a warning at every boot that the file is no executable.

Reply:
Ah, good point! Let's have this optional then!

Reply:
I updated the code and the PR accordingly :wink: 

Thanks!